styles/cart.css

```diff
@@ -0,0 +1,143 @@
+.cart-main {
+    grid-template-columns: 3fr 1.75fr;
+    gap: var(--space-2);
+}
+
+.cart-main-header {
+    border-bottom: 2px solid var(--gray);
+    grid-column: 1 / -1;
+}
+
+.shelf-products {
+    width: 100%;
+    --bg-horizontal-card: #F0f0f0;
```

Comment:
use var here also.

Reply:
Actually I'm setting a variable here locally to the shelf products.